cts/cts-cli.in

```diff
@@ -1856,22 +1856,34 @@ for t in $tests; do
     grep -e '^\* \(Passed\|Failed\)' "$TMPFILE"
 done
 
-if [ $num_errors -ne 0 ]; then
+function print_or_remove_file() {
+
+  eval TMPFILE="\$TMPFILE_$1"
+  if [[ $(diff -wq $test_home/cli/regression.$1.exp "$TMPFILE") ]]; then
+    rm -f "$TMPFILE"
+  else
+    echo "    $TMPFILE"
+  fi
```

Comment:
Forcing a test to fail (by adding a stray character to cts/cli/regression.tools.exp), I realized this does the reverse of what we want. :) It removes the file when there *is* a difference